components/brave_vpn/common/buildflags/buildflags.gni

```diff
@@ -7,3 +7,9 @@ declare_args() {
   # On macOS, vpn is not available w/o signing.
   enable_brave_vpn = is_win || is_android || is_mac
 }
+declare_args() {
+  # On Windows we install vpn service to protect users from DNS leaks.
+  # The service is registered by installer and automatically starts/stops
+  # when BraveVPN connected/disconnected.
+  enable_brave_vpn_service = enable_brave_vpn && is_win
```

Comment:
For me, it's confusing becuase we have `BraveVpnService` class.
How about using different name ex, `enable_brave_vpn_dns_helper_service`?

Reply:
done